vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/impl/BaseTransport.java

```diff
@@ -156,7 +156,11 @@ static void setJSESSIONID(SockJSHandlerOptions options, RoutingContext rc) {
     }
   }
 
-  static void setCORS(RoutingContext rc) {
+  static void setCORS(SockJSHandlerOptions options, RoutingContext rc) {
+    if (!options.isCorsHeadersEnabled()) {
```

Comment:
Sounds nice in theory, but I am not sure how this can be done (I am too new to vertx-web) :)